docs/SPIR-V.rst

```diff
@@ -384,7 +384,38 @@ The ``[]`` operator can also be used to access elements in a matrix or vector. A
 Control flows
 -------------
 
-[TODO]
+Switch Statements
++++++++++++++++++
+
+HLSL switch statements can be translated to SPIR-V in two different ways as detailed in the table below:
+
++-----------------------------+-------------------------------------------------------------------------------------------+
```

Comment:
I feel this table is better replaced with a list:

HLSL switch statements are translated into SPIR-V using

* OpSwitch: if (all case values are integer literals or constant integer variables) and (no attribute or the forcecase attribute is specified)
* A series of if statements: for all other scenarios (e.g., when flatten, branch, or call attribute is specified)